AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt

```diff
@@ -634,12 +635,16 @@ open class DeckPicker :
                 menu.findItem(R.id.action_undo).title = undo
             }
         }
-        updateSearchDecksIconVisibility()
         return super.onCreateOptionsMenu(menu)
     }
 
+    // Try setting it to `col.decks.count() >= 10`. If there was any issue (e.g. corrupt collection), set it to false
```

Comment:
```suggestion
    /**
      * Show [searchDecksIcon] if there are more than 10 decks.
      * Otherwise, hide it if there are less than 10 decks 
      * or if a exception is thrown while getting the decks count (e.g. corrupt collection)
      */
```

Using a javadoc here may be more useful, as the user will be able to see this description while hovering the method on the IDE